spec/models/quiz_spec.rb

```diff
@@ -5,16 +5,38 @@
   it { is_expected.to have_db_index(:course_id) }
   it { is_expected.to validate_presence_of(:title) }
   it { is_expected.to validate_presence_of(:description) }
+  it { is_expected.to validate_presence_of(:passing_percentage) }
   it { is_expected.to validate_presence_of(:questions).
                       with_message('requires at least 1 question.') }
   it { is_expected.to have_many(:exams) }
-  it do
-    is_expected.to have_many(:questions).
-                     order(:created_at).
-                     dependent(:destroy)
+  it { is_expected.to have_many(:questions)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.